aws-lc-rs/src/lib.rs

```diff
@@ -233,14 +233,42 @@ pub fn fips_mode() {
 /// Indicates whether the underlying implementation is FIPS.
 ///
 /// # Errors
-/// Return an error if the underlying implementation is not FIPS, otherwise ok
+/// Return an error if the underlying implementation is not FIPS, otherwise Ok.
 pub fn try_fips_mode() -> Result<(), &'static str> {
     init();
-    unsafe {
-        match FIPS_mode() {
-            1 => Ok(()),
-            _ => Err("FIPS mode not enabled!"),
-        }
+    match unsafe { FIPS_mode() } {
+        1 => Ok(()),
+        _ => Err("FIPS mode not enabled!"),
+    }
+}
+
+#[cfg(feature = "fips")]
+/// Panics if the underlying implementation is not using CPU jitter entropy, otherwise it returns.
+///
+/// # Panics
+/// Panics if the underlying implementation is not using CPU jitter entropy.
+pub fn fips_cpu_jitter_entropy() {
+    try_fips_cpu_jitter_entropy().unwrap();
+}
```

Comment:
Is this normal in Rust to include two functions: one that returns the result and another that unwraps it for the caller?

Reply:
* This change is just following the pattern established w/ `fips_mode()` and `try_fips_mode()`.
* It's fairly typical to name a function "try_xxx" when the operation is fallible. 
* Given that a consumer's compliance status might depend upon FIPS being enabled, having a function that panics the process (and hopefully fails what otherwise would've been a bad/non-compliant deployment) is a convenience.